### PR TITLE
Make footer examples responsive to mobile screens

### LINKED ISSUE #35360: The footer example does not work on mobile
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [x] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

The footer example in https://getbootstrap.com/docs/5.1/examples/footers/ does not work correctly on mobile.
It looks like this on mobile:
a9717491-c75c-441c-82f7-90f3c88ea0cd
53431d5e-a20f-4a1a-b999-257121d53f86

The example should be similar to the main website:
d267e65e-58a4-4d01-b10f-7e4df274b20c

N.B.: The other examples noted in the link above are OK

### Reduced test cases

The footer example in https://getbootstrap.com/docs/5.1/examples/footers/ does not work correctly on mobile.

### What operating system(s) are you seeing the problem on?

Android

### What browser(s) are you seeing the problem on?

Chrome

### What version of Bootstrap are you using?

v5.1.3

### PR DESCRIPTION
**Pull request by nkdas91**
Fixes #35360

### CODE REVIEW

File: site/content/docs/5.1/examples/footers/index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
...
....99 5.827 16 8 16s2.444-.01 3.298-.048c.851-.04 1.434-.174 1.943-.372a3.916 3.916 0 0 0 1.416-.923c.445-.445.718-.891.923-1.417.197-.509.332-1.09.372-1.942C15.99 10.445 16 10.173 16 8s-.01-2.445-.048-3.299c-.04-.851-.175-1.433-.372-1.941a3.926 3.926 0 0 0-.923-1.417A3.911 3.911 0 0 0 13.24.42c-.51-.198-1.092-.333-1.943-.372C10.443.01 10.172 0 7.998 0h.003zm-.717 1.442h.718c2.136 0 2.389.007 3.232.046.78.035 1.204.166 1.486.275.373.145.64.319.92.599.28.28.453.546.598.92.11.281.24.705.275 1.485.039.843.047 1.096.047 3.231s-.008 2.389-.047 3.232c-.035.78-.166 1.203-.275 1.485a2.47 2.47 0 0 1-.599.919c-.28.28-.546.453-.92.598-.28.11-.704.24-1.485.276-.843.038-1.096.047-3.232.047s-2.39-.009-3.233-.047c-.78-.036-1.203-.166-1.485-.276a2.478 2.478 0 0 1-.92-.598 2.48 2.48 0 0 1-.6-.92c-.109-.281-.24-.705-.275-1.485-.038-.843-.046-1.096-.046-3.233 0-2.136.008-2.388.046-3.231.036-.78.166-1.204.276-1.486.145-.373.319-.64.599-.92.28-.28.546-.453.92-.598.282-.11.705-.24 1.485-.276.738-.034 1.024-.044 2.515-.045v.002zm4.988 1.328a.96.96 0 1 0 0 1.92.96.96 0 0 0 0-1.92zm-4.27 1.122a4.109 4.109 0 1 0 0 8.217 4.109 4.109 0 0 0 0-8.217zm0 1.441a2.667 2.667 0 1 1 0 5.334 2.667 2.667 0 0 1 0-5.334z"/>
```
Comment (XhmikosR) [inline site/content/docs/5.1/examples/footers/index.html:18]: This is an unrelated change

Comment (nkdas91) [reply site/content/docs/5.1/examples/footers/index.html]: I added the spaces back. Committed, Rebased and pushed my changes back to the branch. Please check. Thank you!

**Review by ffoodd — APPROVED** [review]
Definitely better, thanks!
